packages/nextjs/src/config/loaders/wrappingLoader.ts

```diff
@@ -202,8 +202,13 @@ export default function wrappingLoader(
       templateCode = templateCode.replace(/__COMPONENT_TYPE__/g, 'Unknown');
     }
 
-    if (sentryConfigFilePath) {
-      templateCode = `import "${sentryConfigFilePath}";`.concat(templateCode);
+    if (sentryConfigFilePath && path.isAbsolute(this.resourcePath)) {
```

Comment:
Why the check for `isAbsolute`?

Reply:
I don't think webpack really defines what the value behind this looks like. *Usually* it is an absolute path but I think it could also be some virtual module and stuff and we can't create relative imports for virtual modules so it is just a check for the cases we can actually handle. Does this make sense?